src/components/Event/EventContainer.tsx

```diff
@@ -0,0 +1,99 @@
+import { useState } from "react";
+import EventBox, { type EventBoxProps } from "./EventBox";
+
+const Faculties = {
+  1: "Engineer",
+  2: "Science",
+  3: "Arts",
+};
+
+const EventContainer = ({ Events }: { Events: EventBoxProps[] }) => {
```

Comment:
please consider move inlining type to a separated prop type
```suggestion
interface EventContainerProps { 
  Events: EventBoxProps[]
}
const EventContainer = ({ Events }: EventContainerProps) => {
```